src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -252,20 +259,21 @@ private Field schemaToField(Schema sch) {
 
   private Message constructEnumMessage(
       String name, String description, List<String> enumVals, List<String> enumDescs) {
-    Message enumMessage = new Message(name, false, true, description);
+    Message enumMessage = new Message(name, false, true, sanitizeDescr(description));
```

Comment:
Your're passing an un-sanitized description to `constructEnumMessage`, but for the Field constructor, you're passing an already-sanitized description. How about adopting a policy that whenever the description is passed as a parameter, it is passed pre-sanitized? That way you can can sanitize descriptions as soon as you get them from an unsanitized source.

Reply:
We can do it but it is not consistent with the rest of the logic. There is currently no preprocessign in the converter, only post-processing.